main.go

```diff
@@ -59,11 +65,43 @@ func init() {
 }
 
 func loadChecks() error {
-	rawChecks, err := ioutil.ReadFile(cfg.CheckDefinitionsFile)
-	if err != nil {
-		return err
+	var (
+		rawChecks []byte
+		err       error
```

Comment:
Why do you declare `err` up here? As far as I can see, it is redeclared/shadowed in ln 73 and ln 86 and then used for the first time in 98, right?